src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -0,0 +1,369 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Getting started with Headless Chrome
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
+{# wf_tags: chrome59,headless,testing #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Learn about Headless Chrome, shipping in Chrome 59. #}
```

Comment:
For the featured snippet maybe you want to hint at why / how peeps use Headless Chrome?